tracer/test/Datadog.Trace.TestHelpers/ProfilerHelper.cs

```diff
@@ -20,7 +20,8 @@ public class ProfilerHelper
             string processToProfile = null,
             bool? enableSecurity = null,
             bool? enableBlocking = null,
-            string externalRulesFile = null)
+            string externalRulesFile = null,
+            int? traceRateLimit = null)
```

Comment:
Similar comment to the one in `EnvironmentHelper.cs`.

Reply:
fixed in 8a1b7d7cfb8567399adc367e4bb2c284b1a35ea7